anesthetic/utils.py

```diff
@@ -225,9 +225,9 @@ def unique(a):
     return b
 
 
-def iso_probability_contours(pdf, contours=[0.68, 0.95]):
+def iso_probability_contours(pdf, contours=[0.95, 0.68]):
     """Compute the iso-probability contour values."""
-    contours = [1-p for p in reversed(contours)]
```

Comment:
I think that we can get round the need for a deprecation warning by putting in a `raise ValueError` statement if the contours are not sorted in the (new) correct order (also in `iso_probability_contours_from_samples`). This will therefore pick up the breaking change in any existing code. Ideally the `ValueError` should also have a note that the breaking change was introduced in version XXX, so that surprised users at least understand where it has come from